Dockerfile

```diff
@@ -116,11 +100,10 @@ RUN pip install matplotlib && \
     # ...as is psycopg2
     apt-get install -y libpq-dev && \
     pip install ibis-framework && \
-    # Cartopy plus dependencies
-    yes | conda install proj4 && \
-    pip install packaging && \
```

Comment:
Are `proj4` and `packaging` going to get installed as dependencies of `Shapely` and `cartopy`?

Reply:
Proj4 definitely is (https://scitools.org.uk/cartopy/docs/latest/installing.html#installing) and interestingly enough so is Shapely - so I assumed from the comment all those  things we were installing were just requirements for Cartopy.

Looking at packaging (https://github.com/pypa/packaging) it doesn't seem like it's something that a user requested and more like it was installed as a dependency - do you have any background on it? (doing git blame didn't help)

Reply:
I moved the conda commands to the top as recommended when using pip alongside conda. So far, it seems like it has solved our flakiness problem: https://github.com/Kaggle/docker-python/pull/760

Once my PR is in, you can merge master and keep your change for `Levenshtein` and `imagemagick` (don't forget to install it at the top along the other conda commands).

Thank you